src/VisualStudio/CSharp/Test/PersistentStorage/AbstractPersistentStorageTests.cs

```diff
@@ -52,7 +51,6 @@ private void CleanUpPersistentFolder()
         {
         }
 
-        [Fact]
```

Comment:
:question: I believe you can leave `[Fact]` here and it will run as an additional test in derived types. Is there a reason you shadowed all the tests instead?